keyboards/atreus62/keymaps/194h/keymap.c

```diff
@@ -0,0 +1,182 @@
+#include "atreus62.h"
+#include "action_layer.h"
+#include "action_util.h"
+
+//The current Nordic and Norwegian layout files are a mess. I'll do it myself:
+//Norwegian
+#define  NO_AE    KC_QUOT     //  Æ
+#define  NO_AO    KC_LBRC     //  Å
+#define  NO_BSLS  KC_EQL      //  Backslash
+#define  NO_LT    KC_NUBS     //  <
+#define  NO_MINS  KC_SLSH     //  -
+#define  NO_OE    KC_SCLN     //  Ø
+#define  NO_PIPE  KC_GRV      //  |
+#define  NO_PLUS  KC_MINS     //  +
+#define  NO_QUOT  KC_NUHS     //  '
+#define  NO_UMLA  KC_RBRC     //  ¨
+//Norwegian - Modifier dependant definitions
+#define  NO_EXLM  LSFT(KC_1)        //  !
+#define  NO_DQT   LSFT(KC_2)        //  "
+#define  NO_AT    RALT(KC_2)        //  @
+#define  NO_HASH  LSFT(KC_3)        //  #
+#define  NO_EUR   LSFT(KC_4)        //  €
+#define  NO_DLR   RALT(KC_4)        //  $
+#define  NO_PERC  LSFT(KC_5)        //  %
+#define  NO_AND   LSFT(KC_6)        //  &
+#define  NO_SLSH  LSFT(KC_7)        //  /
+#define  NO_PO    LSFT(KC_8)        //  (
+#define  NO_PC    LSFT(KC_9)        //  )
+#define  NO_EQL   LSFT(KC_0)        //  =
+#define  NO_LCBR  RALT(KC_7)        //  {
+#define  NO_LBRC  RALT(KC_8)        //  [
+#define  NO_RBRC  RALT(KC_9)        //  ]
+#define  NO_RCBR  RALT(KC_0)        //  }
+#define  NO_COLN  LSFT(KC_DOT)      //  :
+#define  NO_SCLN  LSFT(KC_COMM)     //  ;
+#define  NO_MU    RALT(KC_M)        //  µ
+#define  NO_EURO  RALT(KC_E)        //  €
+#define  NO_SECT  LSFT(KC_GRV)      //  §
+#define  NO_GERC  RALT(KC_COMM)     //  ¸
+#define  NO_QUAR  LSFT(RALT(KC_4))  //  ¼
+//Norwegian layout - dependant on previous definitions
+#define  NO_UNDS  LSFT(NO_MINS)  //  _
+#define  NO_QUES  LSFT(NO_PLUS)  //  ?
+#define  NO_GRAV  LSFT(NO_BSLS)  //  `
+#define  NO_GT    LSFT(NO_LT)    //  >
+#define  NO_HALF  RALT(NO_LT)    //  ½
+#define  NO_HAT   LSFT(NO_UMLA)  //  ^
+#define  NO_ASTR  LSFT(NO_QUOT)  //  *
+#define  NO_TILD  RALT(NO_UMLA)  //  ~
+
+//One Shot Modifier
+#define OSM_LCTL OSM(MOD_LCTL)
+#define OSM_LALT OSM(MOD_LALT)
+#define OSM_LSFT OSM(MOD_LSFT)
+#define OSM_LGUI OSM(MOD_LGUI)
+#define OSM_RCTL OSM(MOD_RCTL)
+#define OSM_RALT OSM(MOD_RALT)
+#define OSM_RSFT OSM(MOD_RSFT)
+#define OSM_RGUI OSM(MOD_RGUI)
+
+//Layers
+#define L1 0
+#define L2 1
+#define L3 2
+#define L4 3
+#define L5 4
+
+// Fillers to make layering more clear
+#define _____ KC_TRNS
```

Comment:
For Lines 69-70, QMK defines keycodes to do this automatically.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

Please delete these lines, and replace all instances of  
`_____` (five underscores) with
`_______` (seven underscores), and all instances of  
`XXXXX` (five Xs) with  
`XXXXXXX` (seven Xs).